src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client.py

```diff
@@ -309,7 +386,7 @@ def _run_single_channel(config: _ChannelConfiguration) -> None:
         channel = grpc.insecure_channel(server)
     with channel:
         stub = test_pb2_grpc.TestServiceStub(channel)
-        futures: Dict[int, Tuple[grpc.Future, str]] = {}
+        futures: Dict[int, Tuple[FutureFromCallType, str]] = {}
```

Comment:
Maybe, instead of fixing `futures` being shadowed in all places, we should just do `import concurrent.futures` instead of `from concurrent import futures` - and use `_global_server = grpc.server(concurrent.futures.ThreadPoolExecutor())`in `_run`.